frontend/src/lib/components/DateFilter/RollingDateRangeFilter.tsx

```diff
@@ -17,8 +17,12 @@ const dateOptions: LemonSelectOptionLeaf<DateOption>[] = [
 ]
 
 type RollingDateRangeFilterProps = {
+    isButton?: boolean
     pageKey?: string
+    // specifies if the filter is selected in the dropdown (to darken)
```

Comment:
As a VS Code user I request using the JSDoc notation (`/** */`), because that gets picked up by VSC for hover tooltips (I think JetBrains is more lax when it comes to JS – I see the point in JSDoc, because it means documentation is separate from notes)